ui/app/components/app/modals/deposit-ether-modal/deposit-ether-modal.component.js

```diff
@@ -88,7 +89,7 @@ export default class DepositEtherModal extends Component {
   render() {
     const { network, toWyre, address, toFaucet } = this.props;
 
-    const isTestNetwork = ['3', '4', '5', '42'].find((n) => n === network);
+    const isTestNetwork = checkTestNetwork(network);
```

Comment:
We should not be using the network ID here - it's deprecated, and will not work correctly if the user interacts with a custom network that has a network ID matching one of our testnets but has a different chain ID.